src/modules/users/users.route.ts

```diff
@@ -69,6 +69,21 @@ router.get('/info', verifyMiddleware.verifyToken, async (req, res) => {
   }
 });
 
+// This endpoint will return a user's information by their ID
+router.get('/info/:id', verifyMiddleware.verifyToken, async (req, res) => {
```

Comment:
should already have the user id saved as a local variable no?

Reply:
ah never mind you need all of the user details?

Reply:
not really, just the first and last name, but i was being lazy